db/schema.rb

```
@@ -12,686 +10,753 @@
+
+  add_foreign_key "addresses", "clients", column: "clients_id"
+  add_foreign_key "cafas_pecfases", "episodes", column: "episode_key", primary_key: "episode_key"
+  add_foreign_key "claim_denied_claim_reasons", "claim_reconciled_claims", column: "claim_record_id", primary_key: "claim_record_id"
+  add_foreign_key "clinical_profiles", "episodes", column: "episode_key", primary_key: "episode_key"
+  add_foreign_key "diagnoses", "episodes", column: "episode_key", primary_key: "episode_key"
+  add_foreign_key "dla20s", "episodes", column: "episode_key", primary_key: "episode_key"
+  add_foreign_key "ite_client_addresses", "ite_client_records", column: "ite_client_id", primary_key: "ite_client_id"
+  add_foreign_key "ite_client_demographic_profiles", "ite_client_records", column: "ite_client_id", primary_key: "ite_client_id"
+  add_foreign_key "ite_client_diagnoses", "ite_client_records", column: "ite_client_id", primary_key: "ite_client_id"
+  add_foreign_key "ite_client_mdm_mappings", "ite_client_records", column: "ite_client_id", primary_key: "ite_client_id"
+  add_foreign_key "ite_client_mdm_mappings", "ite_client_registries"
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)